README.md

```diff
@@ -312,7 +311,6 @@ TOC is generated by [markdown-toc](https://github.com/jonschlinkert/markdown-toc
 - [pulseaudio](https://github.com/andrewrk/pulseaudio): pulseaudio with the build system replaced by zig
 - [libchromaprint](https://github.com/andrewrk/libchromaprint): chromaprint with the build system replaced by zig
 - [raylib](https://github.com/raysan5/raylib/): A simple and easy-to-use library to enjoy videogames programming
-- [openssl](https://github.com/kassane/openssl-zig-zig): TLS/SSL and crypto library (uses Zig Build)
```

Comment:
The correct link is https://github.com/kassane/openssl-zig

Reply:
👍